src/server/plugins/engine/components/UkAddressField.ts

```diff
@@ -186,6 +179,13 @@ export class UkAddressField extends FormComponent {
       }
     }
 
+    if (hint) {
+      hint.id ??= `${name}-hint`
+      fieldset.attributes ??= {
+        'aria-describedby': hint.id
```

Comment:
Here we link the fieldset (by hint ID) so screen readers describe it using the hint text